test/command.js

```diff
@@ -1,3 +1,5 @@
+import _ from 'lodash';
+import Redis from 'ioredis';
 import command from '../src/command';
 
```

Comment:
Add the following to make the `test:jest.js` testing suite pass:

```suggestion
import Redis from 'ioredis';
import command from '../src/command';

// Ensure that we're getting the correct instance of Command when running in test:jest.js, as jest.js isn't designed to test code directly imported private functions like src/command
jest.mock('ioredis', () => {
  const { Command } = jest.requireActual('ioredis');
  const RedisMock = jest.requireActual('../src/index');

  return {
    __esModule: true,
    Command,
    default: RedisMock,
  };
});

```